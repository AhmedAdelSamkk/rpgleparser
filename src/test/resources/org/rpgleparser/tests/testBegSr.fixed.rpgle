     C                   BEGSR     SUBRT1                                      
     C                   ENDSR     SUBRT1                                      
