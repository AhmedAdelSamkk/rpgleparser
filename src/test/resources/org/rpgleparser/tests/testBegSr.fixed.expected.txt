/*===TOKENS===*/
CS_FIXED                 <C>
BlankIndicator           <  >
BlankFlag                < >
BlankIndicator           <  >
CS_BlankFactor           <              >
OP_BEGSR                 <BEGSR>
CS_FactorContent         <SUBRT1>
CS_BlankFactor           <              >
CS_FieldLength           <     >
CS_DecimalPositions      <  >
BlankIndicator           <  >
BlankIndicator           <  >
BlankIndicator           <  >
EOL                      <
>
CS_FIXED                 <C>
BlankIndicator           <  >
BlankFlag                < >
BlankIndicator           <  >
CS_BlankFactor           <              >
OP_ENDSR                 <ENDSR>
CS_FactorContent         <SUBRT1>
CS_BlankFactor           <              >
CS_FieldLength           <     >
CS_DecimalPositions      <  >
BlankIndicator           <  >
BlankIndicator           <  >
BlankIndicator           <  >
EOL                      <
>
/*===TREE===*/
(r
  (cspec_fixed
    C
    (cs_controlLevel '  ')
    (onOffIndicatorsFlag ' ')
    (cs_indicators '  ')
    (factor '              ')
    (cspec_fixed_standard
      (csBEGSR
        BEGSR
        (cspec_fixed_standard_parts
          (factor (factorContent SUBRT1))
          (resultType '              ')
          '     '
          '  '
          (resultIndicator '  ')
          (resultIndicator '  ')
          (resultIndicator '  ')
          \r\n
        )
      )
    )
  )
  (cspec_fixed
    C
    (cs_controlLevel '  ')
    (onOffIndicatorsFlag ' ')
    (cs_indicators '  ')
    (factor '              ')
    (cspec_fixed_standard
      (csENDSR
        ENDSR
        (cspec_fixed_standard_parts
          (factor (factorContent SUBRT1))
          (resultType '              ')
          '     '
          '  '
          (resultIndicator '  ')
          (resultIndicator '  ')
          (resultIndicator '  ')
          \r\n
        )
      )
    )
  )
)
/*======*/